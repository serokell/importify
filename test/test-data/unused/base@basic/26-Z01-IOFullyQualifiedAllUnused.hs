module IOFullyQualfiedAllUnused where

import qualified System.IO

main = return ()
