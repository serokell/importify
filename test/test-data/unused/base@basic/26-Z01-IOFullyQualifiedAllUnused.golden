module IOFullyQualfiedAllUnused where


main = return ()
